yb-voyager/src/metadb/metadataDB.go

```diff
@@ -70,6 +70,11 @@ func CreateAndInitMetaDBIfRequired(exportDir string) error {
 }
 
 func createMetaDBFile(path string) error {
+	err := utils.EnsureDir(path)
```

Comment:
Curious, what prompted you to add this? Any scenario which caused an issue?

Reply:
yeah, in case of other cmds, we have 
```
def CreateMigrationProjectIfNotExists(): 
       which creates all the directory 
       calls initMetaDB(exportDir) -> which internally calls createMetaDBFile(path)
```

So for all cmds except bulk, metainfo directory creation and then initializing MetaDB is taken by that function.

But for bulk command, we need to handle that separately, so i just update this function which is called from `root.go`

Reply:
Ah I see. We are creating metadb ONLY because we want migrationUUID right? 
I guess we could just get a uuid instead of creating metadb, but that's okay. This also works..

Reply:
yes @makalaaneesh , but since `assess-migration-bulk` cmd is retryable, in case of retries/continue-on-error, it better to use same `mig_uuid`

Reply:
Good point, my bad!